### ISSUE TITLE
Voice stops working after first voice command

### ISSUE THREAD

**Issue by PandaNL** [athombv/homey#316]
**What is your Homey version (Settings → System)?**

0.8.22

**What did you try to achieve?**

Use voice command 'Turn on the lights' which turned all my philips hue lights on including my Action wireless switches which also have lights connected to them.

Using other commands like triggering a flow with harmony hub seems to give the same problem.

**What was the result?**

Homey turns on the lights but doesn't listen anymore to 'Hey Homey, or Ok, Homey' after the first command.

A restart is needed to get voice working again.

**Comment by WeeJeWel** [comment]
Could you send a crash log?

**Comment by PandaNL** [comment]
crashlog send: 190B48B324

**Comment by bvdbos** [comment]
confirmed... also after turning lights off : 9C11EF535D

**Comment by bvdbos** [comment]
disabling and re-enabling the microphone clears some queue and let's you give a command once again.

**Comment by WeeJeWel** [comment]
Thanks, on it

**Comment by edtst** [comment]
Confirmed, but inputting commands through speech input website does work.
Log: 9D0A828E06

**Comment by itsamaik** [comment]
Confirmed, also happens after executing "turn the lights on" on the speech input page. 

Log: F68952C988

**Comment by WeeJeWel** [comment]
Thanks, fixed in 0.8.23. This happens after Homey said something.

**Comment by brot123** [comment]
And for me my homey cancel the flow Flow after an ask for confirmation ,the Led ring Goes after the conformation back to rainbow and i have the  one time voice issue too
I have send you an Crash log 4D324310C7

**Comment by bvdbos** [comment]
This works in 0.8.24 but I had to do a system-reset with deleting all settings to get this to work...

**Comment by jghaanstra** [comment]
I can not confirm this. The voice trigger started worked for me after regularly updating and enabling the microphone again. But either way this specific issue is solved.

I did notice I had to change the way I'm saying the trigger to get a good response. I used to articulate very well and had good hit ratio with that but now I have better hit ratio with saying okiyomi fast and as a single word (this was not confirmed by another forum member though).
